### PR TITLE
Transaction ui updates

### PR DESCRIPTION
**Pull request by stackchain**
- Minor updates preparing for the pending transaction feature.

### CODE REVIEW

File: android/hs_err_pid13969.log [hunk 1]
```diff
@@ -0,0 +1,1536 @@
+#
```
Comment (wolverineks) [inline android/hs_err_pid13969.log]: im guessing we should remove this file?

File: src/TxHistory/WalletHero/TabNavigator.tsx [hunk 2]
```diff
@@ -31,7 +31,10 @@ const styles = StyleSheet.create({
   },
   tabText: {
     fontSize: 14,
-    fontWeight: 'bold',
+    fontFamily: Platform.select({
```
Comment (wolverineks) [inline src/TxHistory/WalletHero/TabNavigator.tsx]: why do we need this? they are displaying the same font...

File: src/TxHistory/WalletHero/BalanceBanner.tsx [hunk 3]
```diff
@@ -25,12 +25,19 @@ const styles = StyleSheet.create({
...
+    fontFamily: Platform.select({
+      ios: 'Rubik-Medium',
+      android: 'Rubik-Medium',
+    }),
     color: COLORS.ERROR_TEXT_COLOR_DARK,
   },
   totalText: {
     fontSize: 16,
     lineHeight: 24,
+    fontFamily: Platform.select({
```
Comment (wolverineks) [inline src/TxHistory/WalletHero/BalanceBanner.tsx]: do we need this if they are the same font?

File: src/TxHistory/WalletHero/BalanceBanner.tsx [hunk 4]
```diff
@@ -25,12 +25,19 @@ const styles = StyleSheet.create({
   balanceText: {
     fontSize: 16,
     lineHeight: 24,
-    fontWeight: 'bold',
+    fontFamily: Platform.select({
```
Comment (wolverineks) [inline src/TxHistory/WalletHero/BalanceBanner.tsx]: same

**Review by wolverineks — CHANGES_REQUESTED** [review]
remove error log, update gitignoree

File: src/TxHistory/WalletHero/BalanceBanner.tsx (hunk 3 above)
Comment (stackchain) [reply src/TxHistory/WalletHero/BalanceBanner.tsx]: we don't famous ctrl+c/v, updating

**Review by wolverineks — APPROVED** [review]

**Pull request merged**